pkg/cmd/server/origin/ensure.go

```diff
@@ -73,16 +65,17 @@ func (c *MasterConfig) ensureOpenShiftInfraNamespace() {
 		if err := reconcileRole.RunReconcileClusterRoles(nil, nil); err != nil {
 			glog.Errorf("Could not reconcile %v: %v\n", role.Name, err)
 		}
-
-		addRole := &policy.RoleModificationOptions{
-			RoleName:            role.Name,
-			RoleBindingAccessor: roleAccessor,
-			Subjects:            []kapi.ObjectReference{{Namespace: ns, Name: saName, Kind: "ServiceAccount"}},
+	}
+	for _, roleBinding := range bootstrappolicy.ControllerRoleBindings() {
```

Comment:
We should need any of this moving forward and those are particularly easy to chase. We should stay strict in 3.7